app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person.get("name"), person.get("age"))
+        for person in people
+        if person.get("name") is not None and person.get("age") is not None
+    ]
+    for person in people:
+        if "wife" in person and person.get("wife") is not None:
```

Comment:
Just do like this:
```suggestion
        if person.get("wife") is not None:
```
With husband the same